src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  time: string;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state: State = {
+    time: new Date().toLocaleTimeString(),
+  };
+
+  timerId = setInterval(() => {
+    this.setState({ time: new Date().toLocaleTimeString() });
+    // eslint-disable-next-line no-console
+    console.log(this.state.time);
+  }, 1000);
```

Comment:
Let's do it in componentDidMount() method, to start countdown only after the first render